.vscode/launch.json

```diff
@@ -8,8 +8,7 @@
             "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
             "cwd": "${workspaceRoot}",
             "args": [
-                "-r", "ts-node/register",
-                "source/tests-unit/**/*.ts",
+                "output/tests-unit/**/*.js",
```

Comment:
What was the reason for changing this?  Won't this tell us the line in the JS file that is failing instead of the line in the TS file?